### ISSUE TITLE
Missing textures on quilt 1.20.1

### ISSUE THREAD

**Issue by mintynoura** [ChrysanthCow/epileson#2]
When using Epileson v1.0.1 with Quillted Fabric API 7.5.0+0.91.0-1.20.1 (no other mods), the textures for Epileson armour do not render. The item textures and everything else works fine though

The log reads: 
`[Render thread/WARN]: epileson (anonymous class net.merchantpug.epileson.registry.EpilesonArmorMaterials$1) did not implement getTexture()! Using fallback implementation`

`[Render thread/WARN]: Failed to load texture: minecraft:textures/models/armor/epileson_layer_1.png
java.io.FileNotFoundException: minecraft:textures/models/armor/epileson_layer_1.png
        at (classes and methods that i'm omitting for readability)`
        
`[Render thread/WARN]: Failed to load texture: minecraft:textures/models/armor/epileson_layer_2.png
java.io.FileNotFoundException: minecraft:textures/models/armor/epileson_layer_2.png
        at (more classes and methods)`
        
        
![image](https://github.com/MerchantPug/epileson/assets/107040228/fc243b02-f7cd-4a6c-a5ff-4456beecacd0)

[Full log](https://mclo.gs/qmfj2Z9)

**Comment by ChrysanthCow** [comment]
Okay, Quilt's doing some wacky shit. I'll get an update out to fix this sometime down the line.

**Comment by ChrysanthCow** [comment]
To explain this behaviour, Quilt seems to be fallbacking to the `minecraft` namespace if a QSL armor method isn't overriden, Which is a problem, as I mixin here to use the `epileson` namespace, which Quilt does not respect.

**Comment by mintynoura** [comment]
Yeah that sounds like Quilt lol. Thank you for your reply! I hope it's an easy fix
